stream-chat-android-client/src/main/java/io/getstream/chat/android/client/experimental/socket/Event.kt

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.client.experimental.socket
+
+import io.getstream.chat.android.client.clientstate.DisconnectCause
+import io.getstream.chat.android.client.events.ConnectedEvent
+
+/**
+ * Events
+ */
+internal sealed class Event {
+
+    sealed class Lifecycle : Event() {
+
+        object Started : Lifecycle() {
+            override fun toString(): String {
```

Comment:
Why do we need override `toString()` methods?
On `data class`, can we use the default implementation?

Reply:
Although we don't need to override. But it is mentioned [here](https://github.com/GetStream/stream-chat-android/pull/3561#discussion_r888771110). @kanat any specific reason for that or we can safely remove them?

Reply:
Ok, I think it make sense for `object` but maybe we can keep the `toString()` implementation of `data class`